src/components/ContentUploader/ContentUploader.js

```diff
@@ -480,41 +652,160 @@ class ContentUploader extends Component<DefaultProps, Props, State> {
         }
     };
 
+    /**
+     * Expands the upload manager
+     * 
+     * @return {void}
+     */
+    expandUploadsManager = (): void => {
+        const { windowView, onExpand } = this.props;
+
+        if (!windowView || !onExpand) {
+            return;
+        }
+
+        clearTimeout(this.resetItemsTimeout);
+
+        onExpand();
```

Comment:
Will this ever fail? Thinking if there could be a case where we call this, onExpand fails (or does nothing since it needs to be passed in) and our internal state thinks it is expanded.

Reply:
removing `onExpand`, not needed anymore